### ISSUE TITLE
Get-Secret fails to return password - fields argument cannot contain spaces

### ISSUE THREAD

**Issue by DonPwrShellHunt** [cdhunt/SecretManagement.1Password#7]
```powershell   
 $item = & op get item $Name --fields username, password --vault $VaultName 2>$null | ConvertFrom-Json -AsHashtable
    # fields argument contains a space before password causing the op command to not return password field
    [securestring]$secureStringPassword = ConvertTo-SecureString $item.password -AsPlainText -Force
```

Since $item.password does not exist in returned hashtable, the ConvertTo-SecureString fails

**Comment by cdhunt** [comment]
Fixed in 3653ed1cc15a3afc4e153f260cde725c8ce4c466.

**Comment by DonPwrShellHunt** [comment]
Get-Secret works for me now. Tested Get-Secret for both Login & Password categories on Mac.
Thanks
